tests/components/istio/main_test.go

```diff
@@ -23,18 +16,16 @@ var goDogOpts = godog.Options{
 	Concurrency: 1,
 }
 
-func init() {
-	godog.BindCommandLineFlags("godog.", &goDogOpts)
-}
-
-func TestMain(m *testing.M) {
-	pflag.Parse()
-	goDogOpts.Paths = pflag.Args()
+func InitTest() {
```

Comment:
Go doesn't support running tests with TestMain and go test "-run" parameter, as doing so would cause all of the tests to run either way